src/components/Email/index.ts

```diff
@@ -0,0 +1 @@
+export { Email, EmailProps } from './Email';
```

Comment:
Oh! You exported the Component and the types in one export! Didn’t it complain? I thought you have to prefix with `type` to re-export types.